consul-core/src/test/java/com/smoketurner/dropwizard/consul/core/ConsulAdvertiserTest.java

```diff
@@ -36,12 +24,17 @@
 import com.smoketurner.dropwizard.consul.ConsulFactory;
 import io.dropwizard.jetty.MutableServletContextHandler;
 import io.dropwizard.setup.Environment;
+import org.junit.Before;
+import org.junit.Test;
+
 import java.util.Arrays;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
-import org.junit.Before;
-import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.*;
+import static org.mockito.Mockito.*;
```

Comment:
please don't use * imports and can you also not re-arrange these?

Reply:
I sure can. I will and re-push.